library/spit_results.py

```diff
@@ -234,9 +252,10 @@ def empty_output_message(val, name):
     if JBOSS_EAP_RUNNING_VERSIONS in fact_names:
         val[JBOSS_EAP_RUNNING_VERSIONS] = (
             empty_output_message(
-                safe_ansible_property(host_vars,
-                                      JBOSS_EAP_RUNNING_VERSIONS,
-                                      'stdout'),
+                stdout_err(safe_ansible_property(host_vars,
```

Comment:
There's an issue that was present in the original too, but I just saw it so I'm pointing it out here. `safe_ansible_property` can return None if it doesn't find what it's looking for, and I think this code will ValueError in that case.

We should either change `safe_ansible_property` so it raises an exception if it doesn't find what it's looking for, or handle the None case here.